Pod/Classes/ObjC/URBNValidationRulesCompat.swift

```diff
@@ -115,13 +115,13 @@ import Foundation
     
     /// Here we're making sure that we're given an NSDate because objc is 
     /// dumb and will allow the user to not give an NSDate
-    public override func validateValue(value: AnyObject?) -> Bool {
-        guard let value = value as? NSDate else { return false }
-        return super.validateValue(value)
+    open override func validateValue(_ value: AnyObject?) -> Bool {
+        guard let value = value as? Date else { return false }
+        return super.validateValue(value as AnyObject?)
     }
     
-    public override func validateValue(value: AnyObject?, key: String) -> Bool {
-        guard let value = value as? NSDate else { return false }
-        return super.validateValue(value, key: key)
+    open override func validateValue(_ value: AnyObject?, key: String) -> Bool {
+        guard let value = value as? Date else { return false }
+        return super.validateValue(value as AnyObject?, key: key)
```

Comment:
It's all good. I wasn't exactly articulate in my request. I recalled something about the implicit/explicit casting from 2-3. But since Any works well I wasn't sure if we _should_ do that